modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_4_4/DDMStructureUpgradeProcess.java

```diff
@@ -87,16 +92,19 @@ private void _upgradeDDMStructureVersion() throws Exception {
 				connection.prepareStatement(
 					StringBundler.concat(
 						"select DDMStructureVersion.structureVersionId, ",
+						"DDMStructureVersion.ctCollectionId, ",
 						"DDMStructureVersion.definition from DDMStructure ",
 						"inner join DDMStructureVersion on ",
 						"DDMStructure.structureId = ",
-						"DDMStructureVersion.structureId where ",
+						"DDMStructureVersion.structureId and ",
+						"DDMStructure.ctCollectionId = ",
+						"DDMStructureVersion.ctCollectionId where ",
```

Comment:
Can you please check if this is necessary?

Reply:
Hi @marcelabc !

I removed this code and everything went fine, meaning that the `DDMStructureVersionDDMStructureCTUpgradeProcessTest` passed.